src/components/Dropdown/dropdown.css

```diff
@@ -2,8 +2,55 @@
 @import "src/css/variables/fonts.css";
 @import "src/css/variables/colors.css";
 
-.y-dropdown .ms-Dropdown-label{
+.y-dropdown .ms-Dropdown-label {
   @extend text__mediumSub;
   font-weight: $fontWeight__bold;
   color: $textColor__secondary;
 }
+
+.y-dropdown .ms-Dropdown-title,
+.y-dropdown .ms-Dropdown-caretDownWrapper {
+  height: 36px;
+  line-height: 3.3rem;
+}
+
+.y-dropdown .ms-Dropdown-title {
+  color: $textColor;
+  border-color: $borderColor;
+}
+
+.y-dropdown .y-dropdown__caretDown {
+  color: $color__river;
+}
+
+.y-dropdown .ms-Dropdown:hover .ms-Dropdown-title,
+.y-dropdown .ms-Dropdown:active .ms-Dropdown-title,
+.y-dropdown .ms-Dropdown:focus .ms-Dropdown-title {
+  color: $textColor;
+  border-color: $borderColor;
+}
+
+.y-dropdown .ms-Dropdown:active .ms-Dropdown-title {
+  border-color: $color__lake;
+}
+
+.y-dropdown .ms-Dropdown:focus .ms-Dropdown-title {
+  border-color: $color__lake;
+}
+
+/* NOTE: y-dropdown cannot be used here since the options are not a descendent
+ * of dropdown element
+ */
+.ms-Dropdown-items .ms-Dropdown-item:hover,
+.ms-Dropdown-items .ms-Dropdown-item:active,
+.ms-Dropdown-items .ms-Dropdown-item:focus {
+  background-color: $borderColor__tertiary;
```

Comment:
Should we create some semantic names for$dropdownOption,  $dropdownOption--focused, $dropdownOption--selected ?

Reply:
for borders? i'd say no, probably better to leave them at a slightly higher abstraction so they can be shared by Textfield, etc.

Reply:
should i just leave these as is using $borderColor__tertiary...?

Reply:
that's what i would do. i supposed you could make the argument for `$inputBorderColor` variables, something like:

```
$borderColor: $color__indieRock;
$borderColor__secondary: $color__altRock;
$borderColor__tertiary: $color__popRock;

$inputBorderColor: $borderColor;
$inputBorderColor__focused: $color__lake;
```

Reply:
woops. we're talking border color, but it's background color here, but it's really specific to dropdown options. I think I should just use the colors directly here from the palet for the background colors for these dropdown options.

Reply:
Maybe all those names are too specific. It'd be better to have a more generic name that we can use outside of dropdown.

Reply:
Not that I disagree, but Fabric uses specific colors in their dropdown options from their palette.
```
&:hover {
    background-color: $Dropdown-selectedItem-hover-bg;
    color: $ms-color-black;

    @include highContrastListItemState;
  }

  @include focus-border();

  &:focus {
    background-color: $ms-color-neutralLighter;
  }

  &:active {
    background-color: $Dropdown-selectedItem-hover-bg;
    color: $ms-color-black;
  }

  &.itemIsDisabled {
    background: $ms-color-white;
    color: $ms-color-neutralTertiary;
    cursor: default;
  }
```